campari/model_building.py

```diff
@@ -0,0 +1,537 @@
+# Standard Library
+import warnings
+
+# Common Library
+import galsim
+import numpy as np
+from astropy import units as u
+from astropy.coordinates import angular_separation
+from astropy.utils.exceptions import AstropyWarning
+from erfa import ErfaWarning
+from galsim import roman
+from scipy.interpolate import RegularGridInterpolator
+from snappl.psf import PSF
+
```

Comment:
Clean up import block.  Separate out numpy.  Alphabetize rest.  

`snappl` and `snpit_utils` should be in the same block.